### ISSUE TITLE
[Incorrect result] - Yices return "unsat" on a "sat" file

### ISSUE THREAD

**Issue by numairmansur** [SRI-CSL/yices2#150]
Hi, 
On the following "sat" file, yices returns "unsat".
Z3 and CVC4 correctly return "sat".

[39025_reduced.smt2.zip](https://github.com/SRI-CSL/yices2/files/3859223/39025_reduced.smt2.zip)

`$ yices-smt2 --version`

```
Yices 2.6.1
Copyright SRI International.
Linked with GMP 6.1.2
Copyright Free Software Foundation, Inc.
Build date: 2018-10-26
Platform: x86_64-pc-linux-gnu (release/static)
```

**Comment by ianamason** [comment]
Thanks.  We will look into this.

**Comment by BrunoDutertre** [comment]
Fixed by 931fb4cd554156c009ba8de3.
